src/main/resources/templates/project/projects.html

```diff
@@ -9,22 +9,40 @@
 <div th:replace="fragments/header :: header"/>
 
 <div class="container">
-    <a th:href="@{/projects/add}">
-        <button type="button" class="btn btn-primary">Create project</button>
-    </a>
-    <h1><i class="fas fa-address-book"></i> Projects</h1>
-    <div th:each="project: ${projects}">
-        <div class="card project-card">
-            <img src="https://content.thriveglobal.com/wp-content/uploads/2020/04/0-6.jpeg)"
-                 class="card-img-top" alt="...">
-            <div class="card-body">
-                <h3 class="card-title" th:text="${project.name}"></h3>
-                <p class="card-text" th:text="|Description ${project.projectKey}|"></p>
-                <p class="card-text" th:text="|Project lead ${project.projectLead.firstName} ${project.projectLead.lastName}|"></p>
-                <a th:href="@{/projects/{id}/delete(id = ${project.id})}" class="btn btn-danger">Delete</a>
-                <a th:href="@{/projects/{id}/edit(id = ${project.id})}" class="btn btn-primary">Edit</a>
-            </div>
+    <div class="d-flex flex-row align-items-center">
+        <div class="align-items-left">
+            <h1>
+                <i class="fas fa-address-book"></i> Projects</h1>
         </div>
+        <div>
+            <a th:href="@{/projects/add}">
+                <button type="button" class="btn btn-primary align-items-right">Create project</button>
+            </a>
+        </div>
+    </div>
+
+    <div class="container-table" id="container-table">
+        <table class="table table-sm table-hover bg-secondary">
```

Comment:
use bootstrap list component instead of html table to pass the acceptance criteria